db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_07_25_081029) do
+ActiveRecord::Schema[7.0].define(version: 2023_07_19_150610) do
```

Comment:
why is this included?

Reply:
I don't think there is a 20230725081029 migration, the latest one in the files seems to be 20230719150610, so I'm not sure why 2023_07_25_081029 was committed in the first place. Lmk if I'm missing something tho